contracts/sweep/TokenSweep.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+/**
+ * @title Abstract upgradeable contract that provides the ability to sweep erroneous transferred tokens to a defined beneficiary address
+ *
+ * @dev Access control implementation is required for many functions by design
+ */
+abstract contract TokenSweep is UUPSUpgradeable {
```

Comment:
question (blocking): Is upgradability part of a TokenSweep?

Writing non-upgradabile contracts is acceptable. With `TokenSweep` inherting `UUPSUpgradable` it brings extra baggage that you would not want in that situation (reducing potential code reuse).

Purely from a design PoV, is `UPPSUpgrable` central to what a `TokenSweep` is?  🤔 

_(There are also alternative upgrade patterns e.g. Transparent & Diamond)_

Reply:
Yeah I don't really think it is, considering it's abstract anyway. Would it not make more sense for the inheriting contract to implement the upgradability?

Will remove.

Reply:
> Would it not make more sense for the inheriting contract to implement the upgradability?

I believe so 👍